### PR TITLE
feat: add additional variants of info-box

### PR DESCRIPTION
**Pull request by mert4k**

### CODE REVIEW

File: src/components/info-box/InfoBox.tsx [hunk 1]
```diff
@@ -9,31 +9,37 @@ type VARIANT_TYPES =
...
 }: {
+  fillBg?: boolean;
   icon?: ReactNode;
   variant?: VARIANT_TYPES;
   title: string;
   text: string;
   progressBar?: {
     description?: string;
     level: number;
-    variant?: VARIANT_TYPES;
```
Comment (erdkse) [inline src/components/info-box/InfoBox.tsx:25]: progress bar variant may be different, do we want to keep variants consistent for both boxes and progress bars?

File: src/components/info-box/InfoBox.tsx [hunk 2]
```diff
@@ -9,31 +9,37 @@ type VARIANT_TYPES =
...
 
 export const InfoBox = ({
+  fillBg,
   variant,
   title,
   text,
   progressBar,
   icon,
 }: {
+  fillBg?: boolean;
```
Comment (erdkse) [inline src/components/info-box/InfoBox.tsx:19]: Use full property name please, such as `fillBackground`

File: src/components/info-box/InfoBox.tsx [hunk 3]
```diff
@@ -9,31 +9,37 @@ type VARIANT_TYPES =
...
   text: string;
   progressBar?: {
     description?: string;
     level: number;
-    variant?: VARIANT_TYPES;
   };
 }) => {
   variant = variant || 'primary';
 
+  const iconClassName = 'info-box-icon ' + (fillBg ? '' : ('bg-' + variant));
```
Comment (erdkse) [inline src/components/info-box/InfoBox.tsx:31]: Can we increase readablity of these lines, It is hard to understand what is going on, maybe we can use back quotes (`)

File: src/components/info-box/InfoBox.tsx [hunk 4]
```diff
@@ -9,31 +9,37 @@ type VARIANT_TYPES =
...
+  const infoBoxClassName = 'info-box ' + (fillBg ? ('bg-' + variant) : '');
+
   const progressBarContent = useMemo(() => {
     if (progressBar) {
+      console.log('mert');
```
Comment (erdkse) [inline src/components/info-box/InfoBox.tsx:36]: Can we remove log here?

**Review by erdkse — APPROVED** [review]
lgtm!

**Review by mert4k — COMMENTED** [review]
lgtm

**Pull request merged**